pluto.rosinstall

```diff
@@ -0,0 +1,20 @@
+- git:
+    local-name: pluto_robot
+    uri: git@github.com:uos/pluto_robot.git
+    version: master
+- git:
+    local-name: mesh_navigation
+    uri: git@github.com:uos/mesh_navigation.git
+    version: master
+- git:
+    local-name: mesh_tools
+    uri: git@github.com:uos/mesh_tools.git
+    version: master
+- git:
+    local-name: lvr_ros
+    uri: git@github.com:uos/lvr_ros.git
+    version: master
+- git:
+    local-name: uos_tools
+    uri: https://github.com/uos/uos_tools.git
+    version: kinetic
```

Comment:
The `uos_tools` package is available for rosdep.

Reply:
Thanks! Haven't seen that before. I now removed uos_tools from the rosinstall file.